<strong>Project</strong>: fabric-test</br><strong>Branch</strong>: master<br><strong>ID</strong>: 21651<br><strong>Subject</strong>: [FAB-9964] Change chaincode location<br><strong>Status</strong>: MERGED<br><strong>Owner</strong>: Latitia Haskins - latitia.haskins@gmail.com<br><strong>Assignee</strong>:<strong>Created</strong>: 5/9/2018, 6:07:23 PM<br><strong>LastUpdated</strong>: 5/10/2018, 12:59:51 PM<br><strong>CommitMessage</strong>:<br><pre>[FAB-9964] Change chaincode location

The example chaincode names were renamed to not include
chaincode as the prefix.

Change-Id: I54480cb2edbe62fc4279320eec97bb2a38df2bba
Signed-off-by: Latitia M Haskins <latitia.haskins@gmail.com>
</pre><h1>Comments</h1><strong>Reviewer</strong>: Latitia Haskins - latitia.haskins@gmail.com<br><strong>Reviewed</strong>: 5/9/2018, 6:07:23 PM<br><strong>Message</strong>: <pre>Uploaded patch set 1.</pre><strong>Reviewer</strong>: Hyperledger Jobbuilder - jobbuilder@jenkins.hyperledger.org<br><strong>Reviewed</strong>: 5/9/2018, 6:07:35 PM<br><strong>Message</strong>: <pre>Patch Set 1:

Build Started https://jenkins.hyperledger.org/job/fabric-test-verify-x86_64/1216/</pre><strong>Reviewer</strong>: Scott Zwierzynski - scottz@us.ibm.com<br><strong>Reviewed</strong>: 5/9/2018, 7:25:23 PM<br><strong>Message</strong>: <pre>Patch Set 1:

(1 comment)

LGTM. You fixed the problem, at least mostly. Or maybe all of it (but there is another related problem?) I still see some errors for other reasons, but it seems this is helping. The one I am concerned about is below.  ...chaincode executable file not found...

Error: Could not assemble transaction, err Proposal response was not successful, error code 500, msg error starting container: error starting container: API error (400): OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \\"chaincode\\": executable file not found in $PATH": unknown</pre><strong>Reviewer</strong>: Hyperledger Jobbuilder - jobbuilder@jenkins.hyperledger.org<br><strong>Reviewed</strong>: 5/9/2018, 8:08:39 PM<br><strong>Message</strong>: <pre>Patch Set 1: Verified-1

Build Failed 

https://jenkins.hyperledger.org/job/fabric-test-verify-x86_64/1216/ : FAILURE

No problems were identified. If you know why this problem occurred, please add a suitable Cause for it. ( https://jenkins.hyperledger.org/job/fabric-test-verify-x86_64/1216/ )

Logs: https://logs.hyperledger.org/production/vex-yul-hyp-jenkins-3/fabric-test-verify-x86_64/1216</pre><strong>Reviewer</strong>: Scott Zwierzynski - scottz@us.ibm.com<br><strong>Reviewed</strong>: 5/10/2018, 10:54:38 AM<br><strong>Message</strong>: <pre>Patch Set 1: Verified+1 Code-Review+2</pre><strong>Reviewer</strong>: Scott Zwierzynski - scottz@us.ibm.com<br><strong>Reviewed</strong>: 5/10/2018, 10:54:41 AM<br><strong>Message</strong>: <pre>Removed Verified-1 by Hyperledger Jobbuilder <jobbuilder@jenkins.hyperledger.org>
</pre><strong>Reviewer</strong>: Scott Zwierzynski - scottz@us.ibm.com<br><strong>Reviewed</strong>: 5/10/2018, 10:56:37 AM<br><strong>Message</strong>: <pre>Patch Set 1:

This fixes the bug it was intended to fix, and the tests no longer fail when deploying (installing) chaincode.
There are other failures in the test suite, but we have other bugs opened to fix those, so we can merge this to improve the CI situation.</pre><strong>Reviewer</strong>: Scott Zwierzynski - scottz@us.ibm.com<br><strong>Reviewed</strong>: 5/10/2018, 10:56:40 AM<br><strong>Message</strong>: <pre>Change has been successfully merged by Scott Zwierzynski</pre><strong>Reviewer</strong>: Hyperledger Jobbuilder - jobbuilder@jenkins.hyperledger.org<br><strong>Reviewed</strong>: 5/10/2018, 10:58:56 AM<br><strong>Message</strong>: <pre>Patch Set 1:

Build Started https://jenkins.hyperledger.org/job/fabric-test-merge-x86_64/270/</pre><strong>Reviewer</strong>: Hyperledger Jobbuilder - jobbuilder@jenkins.hyperledger.org<br><strong>Reviewed</strong>: 5/10/2018, 12:59:51 PM<br><strong>Message</strong>: <pre>Patch Set 1:

Build Failed 

https://jenkins.hyperledger.org/job/fabric-test-merge-x86_64/270/ : FAILURE

No problems were identified. If you know why this problem occurred, please add a suitable Cause for it. ( https://jenkins.hyperledger.org/job/fabric-test-merge-x86_64/270/ )

Logs: https://logs.hyperledger.org/production/vex-yul-hyp-jenkins-3/fabric-test-merge-x86_64/270</pre><h1>PatchSets</h1><h3>PatchSet Number: 1</h3><blockquote><strong>Type</strong>: REWORK<br><strong>Author</strong>: Latitia Haskins - latitia.haskins@gmail.com<br><strong>Uploader</strong>: Latitia Haskins - latitia.haskins@gmail.com<br><strong>Created</strong>: 1525903643<br><strong>GitHubRevision</strong>: [fd8dfb14c82532f4a20bd4a99689eec49dda11a5](https://github.com/hyperledger/fabric-test/commit/fd8dfb14c82532f4a20bd4a99689eec49dda11a5)<br><br><strong>Approver</strong>: Scott Zwierzynski - scottz@us.ibm.com<br><strong>Approved</strong>: 5/10/2018, 10:54:38 AM<br><strong>Type</strong>: Code-Review<br><strong>Value</strong>: 1<br><br><strong>MergedBy</strong>: Scott Zwierzynski<br><strong>Merged</strong>: 5/10/2018, 10:56:40 AM<br><br><strong>Approver</strong>: Scott Zwierzynski - scottz@us.ibm.com<br><strong>Approved</strong>: 5/10/2018, 10:54:38 AM<br><strong>Type</strong>: Verified<br><strong>Value</strong>: 1<br><br></blockquote>